ui/src/authors/containers/NumberOfPublishedPapersContainer.jsx

```diff
@@ -0,0 +1,18 @@
+import { connect } from 'react-redux';
+
+import {
+  fetchAuthorPublications,
+  fetchAuthorPublicationsFacets,
+} from '../../actions/authors';
+import LinkLikeButton from '../../common/components/LinkLikeButton';
+
+const PUBLISHED_QUERY = { q: 'citeable:true and refereed:true' };
```

Comment:
Is it possible to move PUBLISHED_QUERY and CITEABLE_QUERY so we can share them with the graph as well?

Reply:
you can move them out in your PR.